src/libcore/lib.rs

```diff
@@ -246,12 +241,9 @@ macro_rules! vector_impl { ($([$f:ident, $($args:tt)*]),*) => { $($f!($($args)*)
 #[path = "../stdsimd/coresimd/mod.rs"]
 #[allow(missing_docs, missing_debug_implementations, dead_code, unused_imports)]
 #[unstable(feature = "stdsimd", issue = "48556")]
-#[cfg(not(stage0))] // allow changes to how stdsimd works in stage0
```

Comment:
Oh these are actually intended to stay as we permanently, currently, don't want to compile stdsimd in stage (allows for us to tweak it more easily)